api/plugins/module_utils/api_client.py

```diff
@@ -316,7 +332,7 @@ def environment_by_id(self, environment_id):
     def environment_get_variables(self, environment):
         query = {
             'query': """query envVars($name: String!) {
-                environmentByOpenshiftProjectName(openshiftProjectName: $name) {
+                environmentByKubernetesNamespaceName(kubernetesNamespaceName: $name) {
```

Comment:
Should we support both queries? I'm not sure how long `environmentByOpenshiftProjectName` will be around for or the plans on deprecating that. We could potentially support both to keep support for older versions of lagoon.

Reply:
Aha, good point. Since I could simply switch `environmentByOpenshiftProjectName` with `environmentByKubernetesNamespaceName` when doing a GraphQL query; I thought that would apply to any Lagoon instance. How would we manage that then? Should we have some kind of `api_version` variable?